packages/ui/src/components/app-sidebar/sidebar-search/sidebar-search.tsx

```diff
@@ -22,36 +22,20 @@ export function SidebarSearch(props: SidebarSearchProps) {
   const { setIsOpen } = searchContext
 
   return (
-    <div>
+    <Layout.Grid gapY="md">
       {props.logo}
```

Comment:
Can we move logo out of search component to sidebar header?

Reply:
Fixed